src/screens/AdminPortal/OrganizationFunds/modal/FundModal.module.css

```
@@ -1,5 +1,93 @@
+  min-height: auto;
+  padding: var(--fund-modal-header-padding-top)
+    var(--fund-modal-header-padding-right)
+    var(--fund-modal-header-padding-bottom)
+    var(--fund-modal-header-padding-left);
+  border-bottom: none;
+}
+
+.createMode :global(.modal-title) {
+  margin: 0;
+  color: var(--color-black);
+  font-family: 'Lato', sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)